client/src/data.js

```diff
@@ -1,6 +1,6 @@
 /* exported data */
 
-let data = {
+export let data = {
   view: 'entry-form',
   entries: [],
   editing: null,
```

Comment:
In React, you will pass `editing` in to the component rather than keeping it in your global `data`.